libs/gl-client/Cargo.toml

```diff
@@ -11,20 +11,25 @@ export = ["chacha20poly1305", "secp256k1"]
 [dependencies]
 anyhow = "*"
 base64 = "^0.21"
+bech32 = "0.9.1"
 bytes = "1.2.1"
 hex = "*"
 http = "0.2"
 http-body = "^0.4"
+lightning-invoice = "0.23.0"
 log = "^0.4"
 pin-project = "*"
 prost = "0.11"
+reqwest = {version="0.11.18", features=["json"]}
```

Comment:
This is implicitly pulling in `openssl-sys` which depends on the openssl headers. We already use `rustls` through `tonic`, so we likely want to use the `rustls-tls-native-roots` feature.